### PR TITLE
[#272] Added a color theme & font to home page

### PR DESCRIPTION
**Pull request by Omoorion**

### CODE REVIEW

**Review by Monkey-bored — COMMENTED** [review]
I am not convinced by this font

File: core/styles/components.css [hunk 1]
```diff
@@ -251,3 +250,14 @@ input#shop-upgrade-swapper {
...
   padding-bottom: 5%;
 }
+
+button {
+  border: none;
+  outline: none;
+  border-radius: 1em;
+  padding: 1em;
+  font-weight: bold;
+  font-family: 'Franklin Gothic Medium';
```
Comment (Ido-Barnea) [inline core/styles/components.css:260]: code duplication with `base.css`

File: core/styles/base.css [hunk 2]
```diff
@@ -15,4 +15,5 @@ body {
   flex-direction: row;
   justify-content: center;
   align-items: center;
+  font-family: 'Franklin Gothic Medium';
```
Comment (Ido-Barnea) [inline core/styles/base.css:18]: Why this font?

**Review by Ido-Barnea — CHANGES_REQUESTED** [review]

**Comment by Ido-Barnea** [comment]
Tested your changes locally @Omoorion , please consider doing the following:
1. The "Join Game" button's radius is a bit too big, please change it `8px`.
2. This is not related to this issue but is a fix to the previous one. Please make the border color of the `logs-container` and the `shop-upgrade-swapper` `white`.
3. And again, please revert to the previous font used by the website. The font is `system-ui, sans-serif`.

**Pull request merged**